### ISSUE TITLE
db not saved to file

### ISSUE THREAD

**Issue by fvea** [yahoo/redislite#39]
Redislite is not dumping to file for me:

```
>>> from redislite import Redis
>>> r = Redis('test.db')
>>> r.set('foo', 'bar')
>>> r.get('foo')
bar
```

```
>>> from redislite import Redis
>>> r = Redis('test.db')
>>> r.keys()
[]
```

Python 2.7.9 on Yosemite, installed via pip install redislite.

**Comment by dwighthubbard** [comment]
I'm pretty sure this is due to the default working directory being the temp directory redislite creates for the configuration, so giving it a db name without a path element is creating the db in the temp directory that gets cleaned up when the object gets garbage collected or python exits.

From your problem description I think we should change this behavior.  I'm going to add this issue to the redislite sprint at pycon but may get to it before then.

**Comment by dwighthubbard** [comment]
Submitted pull request #40 to fix this issue.

**Comment by dwighthubbard** [comment]
The fix for this is in the 1.0.133 version of redislite.
